client/src/pages/Map/MapLayers.js

```diff
@@ -4,31 +4,17 @@ import { treeHealthUtil } from '@/util/treeHealthUtil';
 import TreeCountLayer from './TreeCountLayer';
 import TreeLayer from './TreeLayer';
 
-// TODO make the dots bigger on zoom out so you can see which cities have trees
-const circleRadius = {
-  'circle-radius': [
-    'interpolate', ['linear'], ['zoom'],
-    10, 2,
-    11, 3,
-    17, 9,
-  ],
-};
-const circleLayerZoomRange = {
-  type: 'circle',
-  minzoom: 5,
-  maxzoom: 18,
-};
+const linearZoom = ['interpolate', ['linear'], ['zoom'],
+  5, 10,
+  7, 7,
+  10, 2,
+  18, 6,
```

Comment:
Why is the radius getting smaller and then bigger as the zoom goes up?  And why are the circles visible so far out?  Shouldn't just the city names be visible at that zoom level?